scout/dashboard.md

```diff
@@ -79,6 +79,21 @@ Each entry in the list shows the following details:
 - The package type.
 - The number of images that use the package.
 
+### Images using base image
+
+![A screenshot of the Docker Scout view showing images that use a selected base image](./images/dashboard-images-used-by.png)
+
+The **Images using base image** tab shows all images in an organization that use a specific base image.
+
+Each entry in the list shows the following details:
```

Comment:
**suggestion:** could we potentially talk about how we get to this page? As in, from the images page